x-pack/legacy/plugins/code/public/components/diff_page/diff.scss

```diff
@@ -1,16 +1,52 @@
+.codeDiffParentCommits {
```

Comment:
Can you try to use BEM classes here? Mainly I think `codeDiff` is the block in most of these classes, so we just need to add some underscores for better legibility.

Reply:
This is useless, too, just delete this